connutils_test.go

```diff
@@ -437,7 +437,7 @@ func TestConnectTimeout(t *testing.T) {
 		User:               opts.User,
 		Password:           opts.Password,
 		Database:           opts.Database,
-		ConnectTimeout:     1 * time.Millisecond,
+		ConnectTimeout:     2 * time.Nanosecond,
```

Comment:
Why 2ns and not 1ns? :)

Reply:
The ConnectTimeout should be longer than WaitUntilAvailable so that the timeout is caused by the ConnectTimeout duration and not the WaitUntilAvailable duration.